python/doc/examples/functional_modeling/vectorial_functions/plot_createMultivariateFunction.py

```diff
@@ -0,0 +1,168 @@
+"""
+=============================
+Create multivariate functions
+=============================
+We can create multivariate functions by different methods. One of the methods is to gather
+multivariate functions. The other one is to create a function having multivariate input using
+tensorization of univariate functions. In this example, we present both methods.
+"""
+# %%
+#
+# Import the required modules
```

Comment:
This is an unnecessary comment: otherwise, any example would start with this statement.
```suggestion
```

Reply:
ok